Rakefile

```diff
@@ -0,0 +1,87 @@
+# frozen_string_literal: true
+
+require 'puppet_litmus/rake_tasks' if Bundler.rubygems.find_name('puppet_litmus').any?
```

Comment:
Same here, I think https://github.com/puppetlabs/puppetlabs-terraform/blob/main/Rakefile are the only tasks we actually use and we manually update the CHANGELOG - the autogenerated ones tend to be not great, and I don't expect this to get released super often.